src/main/java/gyro/google/compute/ResourcePolicyFinder.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.ResourcePoliciesScopedList;
+import com.google.api.services.compute.model.ResourcePolicy;
+import com.google.api.services.compute.model.ResourcePolicyAggregatedList;
+import com.google.api.services.compute.model.ResourcePolicyList;
+import gyro.core.Type;
+import gyro.google.GoogleFinder;
+
+/**
+ * Query for resource policies.
+ *
+ * Example find the address filtering on name equal to 'us-east1-test-two' in the region 'us-east1'.
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    policies: $(external-query google::compute-resource-policy { filter: 'name = "example-policy-alpha"', region: 'us-central1' })
+ *
+ */
+@Type("compute-resource-policy")
+public class ResourcePolicyFinder extends GoogleFinder<Compute, ResourcePolicy, ResourcePolicyResource> {
+
```

Comment:
Field `filter` and `region` and corresponding getters and setters.

The getters will have the docs that describes the fields, and later we will be changing the usage to use the fields and not the map.

Reply:
Done.